15-shell-commands.md

```diff
@@ -33,7 +35,7 @@ Therefore this must be kept in mind if other proprietary apps wants to access th
 
 ## Setting shell to default from any app
 
-If you want create a smart app that will communicate with RFC any time, there is a sequence that resets the shell to its default state under any condition (no need to cycle USB). 
+If you want create a smart app that will communicate with RFC any time, there is a sequence that resets the shell to its default state under any condition \(no need to cycle USB\).  
```

Comment:
This may be to do with the editor you use, but you  don't need to escape characters with a \